### DISCUSSION TITLE
a option to disable RDP Session Sharing in Console strategy settings (for servers) + quick question about updates

### DISCUSSION THREAD

**Discussion by IzunaMoon** [rustdesk/rustdesk-server-pro#486] [General]
We update the RustDesk client on our servers / laptops using a script from the RustDesk documentation. This script works great as it allows us to automatically update all our PCs, eliminating the need to manually take over each PC to update RustDesk.

We’re encountering an issue due to a recent behavior change in RustDesk. When RDP session sharing is enabled, we cannot access our servers over RDP unless someone is already logged in. This makes it difficult to manage or access our servers sins they are servers rdp u-is already enable in windows server.

Would it be possible to add an option in Device Strategies to disable RDP session sharing? This way, we wouldn’t need to manually disable it each time RustDesk updates itself sins it seems to re enable it after every update.

I have a minor question regarding updates. I noticed you added a button to notify users that they need to update their RustDesk client. Are there any plans to introduce an option for the client to update itself automatically, similar to other remote control services? This would eliminate the need for our script entirely.

Currently, the script works well within our domain, but we still have around 70 clients that require manual updates because the script needs admin permissions. We run the script under the system account, so it has the necessary permissions, but automatic updates would greatly simplify the process.

This challenge is making it difficult for us to continue with RustDesk, especially considering the recent price increase. Some competitors are becoming more attractive price-wise due to their lower maintenance requirements for IT departments. We prefer to run the latest version of our software, but manually handling updates isn't always feasible.

**Comment by 21pages** [comment]
Thanks for the feedback! 

These are good suggestions about RDP session sharing control and auto-update features, though they're not high priority items right now.

For now, you might want to keep using your update script as it seems to work well for most of your setup.

**Reply by IzunaMoon** [reply]
We’ve been using your software for a year now, and this is the only major issue we’ve encountered. Our antivirus, Webroot, recently started flagging older versions of RustDesk. To address this, we took your script and made some minor alterations.

We just wanted to let you know our biggest concerns with RustDesk right now:
	1.	Lack of automatic updates
	2.	RDP session sharing interfering with RDP access

When RDP session sharing is enabled, we can’t access our servers via RDP unless someone is already logged in. We need to log directly into the server, disable the feature, and then everything works as expected. For now, I might revise the script so it doesn’t update our servers automatically, and instead, we’ll manually update the servers until these issues are resolved.

We’re very happy with RustDesk overall. Having a self-hosted solution where all data remains within our company is a big advantage. Only authorized companies can access our machines directly via our servers, which is exactly what we need.

If these two issues can be resolved in the next few months, we’ll be very pleased, as it will allow us to treat RustDesk as a self-maintaining solution. This would also reduce the stress we faced recently when our antivirus deleted over 300 installations 2-3 weeks ago.

We haven’t raised these issues before since they didn’t affect us previously, but we genuinely want to see RustDesk grow and become even better. In our eyes, RustDesk has the potential to surpass many other solutions that offer limited control over the client.

Please continue the great work you’re doing. We’re very supportive and interested in how RustDesk will develop further. We also hope you’ll consider not increasing the price further or perhaps introduce a non-profit pricing plan, as the recent price increase was quite steep for us at renewal.

Overall, we are happy with the software and appreciate all the hard work the RustDesk team puts into the product. Thank you!
